cypress/integration/Crosscutting/PublicUseTest/index.js

```diff
@@ -0,0 +1,33 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('user is navigating to {string}', (a) => {
```

Comment:
Nadia is creating a ticket to keep track of all the tests that use workaround - please reference this in there, since we want to remove it once the fix makes it to the platform